docs/grid/custom_filter.md

```diff
@@ -4,6 +4,124 @@ Sylius Grids come with built-in filters, but there are use-cases where you need
 
 To add a new filter, we need to create an appropriate class and form type.
 
+## Using attributes
+
+{% code title="src/Grid/Filter/SuppliersStatisticsFilter.php" lineNumbers="true" %}
+```php
+<?php
+
+namespace App\Grid\Filter;
```

Comment:
Since it's a full php code and not just snippet -- I think we should use strict over here.

```suggestion
<?php

declare(strict_types=1);

namespace App\Grid\Filter;
```